docs/src/overview/get-support.md

```diff
@@ -0,0 +1,40 @@
+---
+title: " Get support"
+weight: 4
+toc: false
+description: |
+  Find out how to get help If you're experiencing an issue with Platform.sh.
+---
+
+{{< description >}}
+
+
+## Create a support ticket
+
+1. [Open the Management Console](https://console.platform.sh/)
+2. Select **User menu** (your name) > **Support**.
+3. Click **+ New ticket**.
+4. Fill in the ticket fields and click **Submit**.
+
+In the future, you can [access the ticket list](https://console.platform.sh/-/users/~/tickets) or [open a ticket directly](https://console.platform.sh/-/users/~/tickets/open) using the shortcuts.
+
+Once you submit a ticket, you can see it on the list alongside tickets created by users in your organization over time.
+Note that once you submit the ticket you won't be able to modify or delete the submission.
+If you have any additional information, you can select the submitted ticket and write a message.
+
+
+
+## Slack
+
+If you want to chat with engineers and other customers you can join the [public Slack channel](https://chat.platform.sh/).
```

Comment:
```suggestion
To chat with engineers and other customers, join the [public Slack channel](https://chat.platform.sh/).
```
Again, I think chatting doesn't match the experience the user is probably currently having. It feels too positive and not focused on a resolution. What kinds of questions should they bring there? Or when to go there? The idea is that there should be some sort a decision tree for deciding which channel to take when I have an issue. So something that coordinates with the reasoning for a support ticket.

Reply:
As I mentioned above based on a conversation with Support. They say:
"That is a great question! And I don’t think there is a well defined answer to it.
Support team does monitor the channel to make sure any problems customers might bring up there are addressed. But at the same time. we don’t really like to turn it into a support ticket escalation channel, as there’s no SLA for Pro support requests.
Lot of the others who answer more application or framework related questions do so voluntarily based on their own expertise. Not sure we’ve ever agreed on the real goal of the channel, some hate it, some think it’s a necessary evil, but I don’t think anyone considers it an ideal solution for community based knowledge sharing or help."